### ISSUE TITLE
set up mechanical streams

### ISSUE THREAD

**Issue by dustymabe** [coreos/fedora-coreos-tracker#180]
The mechanical streams (as referenced by https://github.com/coreos/fedora-coreos-tracker/blob/master/stream-tooling.md) need to be set up so that we can then use their output (lockfiles as input) to our development and production refs. Let's set them up and identify new issues along the way so that we can unblock our production refs.

**Comment by dustymabe** [comment]
related: https://github.com/coreos/fedora-coreos-config/issues/100

**Comment by dustymabe** [comment]
@jlebon - is this one mostly done now?

**Comment by jlebon** [comment]
It's done for `bodhi-updates`. Could probably leave it open to track adding the other mechanical streams and hooking up their lockfiles.

**Comment by dustymabe** [comment]
I'd say while we are still wiring things up the other mechanical streams will be a distraction (for preview). Maybe let's close this out and create a new ticket for the other streams (for after preview).

**Comment by jlebon** [comment]
Sure, SGTM!
